qa/automation/CareerPortal.Tests.Scenarios/Career Portal/Send your CV.feature

```diff
@@ -0,0 +1,170 @@
+	
+	Examples:
+	|format| size |  status  |
+    | PDF  |10,1Mb|  failed  |
+	| DOC  | 10Mb |successful|
+	| DOCX | 9,9Mb|successful|
+	| RTF  | 1Mb  |successful|
+	| PNG  | 8Mb  |  failed  |
+	| JPEG | 11Mb |  failed  |
+
+ Scenario: 4.3 "Attach CV" button when valid file is attached
+	When User attaches a valid CV file
+	Then "Attach CV" button should turn green
+	And "Attach CV" buttion should be still available for User to click
+
+Scenario: 4.4 Changing CV file for a new one
+	Given User has attached a valid CV file 
+	When User clicks "Attach CV" button 
+	Then Then modal window with available <format> device files opens
+	
+	Examples:
+	|format|
+    | PDF  |
+	| DOC  | 
+	| DOCX |
+	| RTF  | 
+
+Scenario: 5.1 "Send" button if all mandatory requirements are met
+	Given User has correctly filled in the "Name" and "Email" text fields
+	And User has attached a CV file
+	And User has checked "I agree to the personal data processing" checkbox
+	When User clicks the Send button
+	Then CV file sent successfully 
+
+Scenario: 5.2.1 "Send" button without filled in the "Name" texted field
+	Given User has correctly filled in the "Email" text field
+	And User has attached a CV file
+	And User has checked "I agree to the personal data processing" checkbox
+	But User has not filled in the "Name" text field
+	When User clicks "Send" button
+	Then red "This field is required" message box should appear under the "Name" texted field
+
+Scenario: 5.2.2 "Send" button without filled in the "Email" texted field
+	Given User has correctly filled in the "Name" text field
+	And User has attached a CV file
+	And User has checked "I agree to the personal data processing" checkbox
+	But User has not filled in the "Email" text field
+	When User clicks "Send" button
+	Then red "This field is required" message box should appear under the "Email" texted field
+
+Scenario: 5.2.3 "Send" button without attached CV file
+	Given User has correctly filled in the "Name" and "Email" text fields
+	And User has checked "I agree to the personal data processing" checkbox
+	But User has not attached a CV file
+	When User clicks "Send" button
+	Then text "Attach CV" inside the "Attach CV" button should turn red
+
+Scenario: 5.2.4 "Send" button without checked "I agree to the personal data processing" checkbox
+	Given User has correctly filled in the "Name" and "Email" text fields
+	And User has attached a CV file
+	But User has not checked "I agree to the personal data processing" checkbox
+	When User clicks "Send" button
+	Then text "I agree to the personal data processing" should turn red
+
+Scenario: 5.2.5 "Send" button without filled in "Message" text field
+	Given User has correctly filled in the "Name" and "Email" text fields
+	And User has attached a CV file
+	And User has checked "I agree to the personal data processing" checkbox
+	But User has not filled in "Message" text field
+	When User clicks the Send button
+	Then CV file sent successfully
```

Comment:
In the last part, you could also try to enter invalid values into various fields and use pairwise(because here you checked only singular parts- name,email, CV attach, checkbox - you can also combine them:) also I think that Scenario№5.1 and Scenario№5.2.5 the same, maybe you missed line about "Message"

Reply:
Can't quite agree with scenario 5.1 and 5.2.5 point. 
5.1 - that "send" button works correctly when user has filled in all the required fields,
5.2.5 - that "send" button works, even if user does not fill in an optional field.
I could be totally wrong here but for me it feels like two different scenarios. 


P.S. Thank you for your review, it was very useful to me!